test/integration/scheduler/predicates_test.go

```diff
@@ -1061,6 +1090,8 @@ func TestUnschedulablePodBecomesSchedulable(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			testCtx := initTest(t, "scheduler-informer")
 			defer testutils.CleanupTest(t, testCtx)
+
+			nodeName = "node-" + uuid.New().String()
```

Comment:
one preference on my side would be to allow the `init` and `update` functions to accept `cs kubernetes.Interface, nodeName string`. the node name would then be fed on each invocation of these functions and not declared outside of the test cases as shared (line 1043).

Reply:
Yeah, thinking about it this solution would be cleaner.

Originally I didn't go for this solution, because (as far as I can tell), all the tests that I am planning to add use a single node created in the `init` function and only this specific test updates the node. My concern was that we might bloat `init` and `update` unnecessarily. The code is clearer this way though.